submissions/YuliiaHordiichuk/memory-game/index.js

```diff
@@ -0,0 +1,91 @@
+class Game {
+    constructor() {
+        this.cardsContainer = document.querySelector('.cards'); 
+        this.cardsBack = ['cat', 'dog', 'elephant', 'fox', 'jaguar', 'zebra'];
+        this.matchCards = 0; 
+        this.winMatchCards = 6; 
+        this.oneMatchCards = 2; 
+        this.openCards = []; 
+        this.delayToCloseCards = 400; 
+        this.delayToHideCards = 400; 
+        this.delayToAlertWin = 600;
+
+        this.fillCards();
+        this.addClickHendler();
+    }
+
+    fillCards () {
+        this.cardsContainer.innerHTML = this.createCards();
+    }
+
+    mixCards () {
+        return this.cardsBack.concat(this.cardsBack).sort(() =>  0.5 - Math.random());  
+    }
+
+    createCards () {
+        return this.mixCards().map(card => {
+            return `
+            <li class='card'>
+                <button class='card__btn' data-id='${card}'>
+                    <img class='card__front' src='assets/cards-front.jpg' alt='card-front'>
+                    <img class='card__back' src='assets/${card}.jpg' alt='card-back'>
+                </button>
+            </li>`
+        }).join(''); 
+    }; 
+
+    addClickHendler () {
+        this.cardsContainer.addEventListener('click', ({ target: button }) => {
+            const buttonId = button.dataset.id;
+
+            if(buttonId) {
+                button.classList.add('active');
```

Comment:
Again mix of single quotes and double quotes. No matter what you choose, just be consistent.